generate_collection_of_logs.py

```diff
@@ -82,8 +82,8 @@ def generate_logs(file_path_to_own_models=None):
         # ADD NOISE and CREATE NOISE INFO INSTANCE
         noise = select_random(par.Noise, option='random')
```

Comment:
Is the select_random for the noise variable really necessary? Isnt it just the value of par.Noise anyway?